content/api/_index.md

```diff
@@ -6,42 +6,42 @@ weight: 1
 Programming your robot is done in [Python](https://www.python.org/), specifically version {{% python_version %}}. You can learn more about Python from their [docs](https://docs.python.org/{{% python_version %}}/), and our [whirlwind tour](/tutorials/python-whirlwind-tour).
 
 ## Setup
-The following two lines are required to setup the robot.
+The following two lines are required to initialise the robot.
 
 ```python
 from robot import Robot
 
 r = Robot()
 ```
 
-This will setup your robot and allow it to send and receive data from the robot's various boards. Once this has been setup, this object can be used for most of the functions of the robot.
+This will initialise your robot and allow it to communicate with the robot's various boards. Once this has been done, this `Robot` object can be used for most of the functions of the robot.
 
 The remainder of the tutorials pages will assume your robot is defined as `r`.
 
 ## Where to put your code
-Your code needs to be put on a USB drive in a file called `main.py`. On insertion, this file will be executed. The file is directly executed off your USB drive, with your drive as the working directory.
+Your code needs to be put on a USB drive in a file called `main.py`. On insertion into the robot, this file will be executed. The file is directly executed off your USB drive, with your drive as the working directory.
 
 {{% notice warning %}}
 If this file is missing or incorrectly named, your robot won't do anything. No log file will be created.
 {{% /notice %}}
 
 ## Start Button
-So your robot doesn't start moving as soon as the robot is finished setting up, the code is paused after setup until the _Start Button_ on the power board is pressed. There is a green LED next to the start button when the robot is finished setting up and the start button can be pressed.
+After the robot has finished starting up, it will wait for the _Start Button_ on the power board to be pressed before running your code, so that you can control when it starts moving. There is a green LED next to the start button which flashes when the robot is finished setting up and the start button can be pressed.
```

Comment:
Is this strictly true? The definition of "your code" as implied above is anything in the `main.py`. Surely it's actually paused inside the call to `Robot()`?

Reply:
Changed "running your code" to "continuing with your code".